draft-ietf-quic-datagram.md

```diff
@@ -206,20 +206,33 @@ and can store the contents in memory.
 
 DATAGRAM frames MUST be protected with either 0-RTT or 1-RTT keys.
 
-Application protocols using datagrams are responsible for defining the
-semantics of the Datagram Data field, and how it is parsed. If the application
-protocol supports the coexistence of multiple entities using datagrams inside
-a single QUIC connection, it can use a mechanism to allow demultiplexing
-between them. For example, using datagrams with HTTP/3 involves prepending
-a flow identifier to all datagrams, see {{?I-D.schinazi-quic-h3-datagram}}.
-
 Note that while the max_datagram_frame_size transport parameter places a limit
 on the maximum size of DATAGRAM frames, that limit can be further reduced by
 the max_packet_size transport parameter, and by the Maximum Transmission Unit
 (MTU) of the path between endpoints. DATAGRAM frames cannot be fragmented,
 therefore application protocols need to handle cases where the maximum datagram
 size is limited by other factors.
 
+## Multiplexing Datagrams
+
+DATAGRAM frames belong to a QUIC connection as a whole, and are not strongly
+associated with any stream ID at the QUIC layer. However, it is expected
+that applications will want to differentiate between different DATAGRAM frames
```

Comment:
"differentiate between different" seems kind of redundant. I tried penning a suggestion for this sentence but it didn't seem to read any better

Reply:
```suggestion
that applications will want to differentiate between specific DATAGRAM frames
```